point_cloud_transport/include/point_cloud_transport/point_cloud_transport.hpp

```diff
@@ -126,15 +138,44 @@ Subscriber create_subscription(
   const std::string & transport,
   rmw_qos_profile_t custom_qos = rmw_qos_profile_default,
   rclcpp::SubscriptionOptions options = rclcpp::SubscriptionOptions());
+POINT_CLOUD_TRANSPORT_PUBLIC
+Subscriber create_subscription(
```

Comment:
Add documentation too

```suggestion
  rclcpp::SubscriptionOptions options = rclcpp::SubscriptionOptions());

POINT_CLOUD_TRANSPORT_PUBLIC
Subscriber create_subscription(
```